### ISSUE TITLE
[Google Chrome] Automatically sign in for basic authentication doesn't work anymore

### ISSUE THREAD

**Issue by DoCode** [bitwarden/browser#696]
# Steps to reproduce

Credentials for a web page with basic auth exists in Bitwarden.  
Open a web page with basic auth enabled.

# Expected behavior

Bitwarden sends the credentials to the web page automatically.

# Actual behavior

Enter credentials popup from GC is displayed.

# Environment data

- Google Chrome: 69.0.3497.32 (Official Build) beta (64-bit)
- Bitwarden: 1.29.0

**Comment by kspearrin** [comment]
Do you have more than 1 matching login for that URL?

**Comment by DoCode** [comment]
Yes, multiple. Mapped with the Host matching-pattern, because it's all subdomains.

**Comment by kspearrin** [comment]
Basic auth will only auto-login if there is only 1 matching login for that website. Otherwise we don't know which one you want to use.
